.github/workflows/setup-python-environment.yml

```diff
@@ -1,4 +1,4 @@
-name: Setup the Python Environment
+name: Setup Python Environment, Lint and Unit Test
```

Comment:
I would not touch this, cause it is used elsewhere (like inside Integration tests), so I'd better repoint CI jobs to `lint-unit-test.yaml` for now and keep this untouched (or, better, revisit how it is used and adjust).
I'm pretty sure that with this setup Integration tests are also running a full set of units (see step `Setup environment` there)

Reply:
My bad: misread the structure of actions/scripts. All good here